tests/if/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -31,3 +31,19 @@ end
 puts 'this is true' if true
 
 `;
+
+exports[`if_3.rb 1`] = `
+def foo()
+  true
+end
+
+if foo()
+  puts "this is true"
+end
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+def foo
+  true
+end
+puts 'this is true' if foo()
```

Comment:
I decided to only fix the crash, rather than trying to pretty this up and drop the unneeded parens as well.

Reply:
Can you pretty this up too? so that we can get this merged?